paz/models/classification/cnn2Plus1.py

```diff
@@ -0,0 +1,354 @@
+import einops
```

Comment:
This import should be removed. It could raise an error to all users that don't have this.

Reply:
Oh, I'm sorry. I had those changes shelved and forgot to commit them.